packages/terra-pills/src/index.js

```diff
@@ -0,0 +1,3 @@
+import FilterPills from './FilterPills';
+
+export default FilterPills;
```

Comment:
Should we rename this package from terra-pills to terra-filter-pills because the default export is named `FilterPills`?

Reply:
After further discussion with the team FilterPills is just one variant of pills and we may have other variants in the future. For this reason, keeping terra-pills as the package name make the most sense. FilterPills can be updated from the default export to be a named export in the future if necessary but for now we'll keep it the way it is.